ci-operator/step-registry/aws/deprovision/route53/OWNERS

```diff
@@ -0,0 +1,6 @@
+approvers:
+- patrickdillon
+- wking
```

Comment:
I am no longer tied into AWS enough to be a useful maintainer here.  Can you drop me?

Reply:
Done